### PR TITLE
Update 2.8 docs to 2.8.7

### PR DESCRIPTION
**Pull request by simonmaxen-da**

### CODE REVIEW

**Review by carrielaben-da — APPROVED** [review]

File: docs/2.8.7/versions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "daml": "2.8.6",
```
Comment (yves-da) [inline docs/2.8.7/versions.json:2]: This should be `2.8.7` for daml and canton, no?

**Review by yves-da — CHANGES_REQUESTED** [review]
Please double-check the versions in the versions.json file. Is it as intended?

File: docs/2.8.7/versions.json (hunk 1 above)
Comment (garyverhaegen-da) [reply docs/2.8.7/versions.json:2]: No, component versions here. We have a Daml Enterprise 2.8.7, but sdk, canton, and drivers are all still at 2.8.6.

**Review by yves-da — APPROVED** [review]
@simonmaxen-da I'll approve the PR tentatively, because I don't want to block you. Please have a look at my comment, though.

**Pull request merged**